CONVENTIONS.md

```diff
@@ -306,6 +306,18 @@ the Prometheus instance in the cluster to query the
 `kube_pod_resource_request` and `kube_pod_resource_limit` metrics and
```

Comment:
should we remove `kube_pod_resource_limit` since it is stated before that cluster components shouldn't have resource limits?

Reply:
We could do that in another PR, if we're sure nothing actually does set limits.